src/test-helpers/e2e-helpers/getEditingText.ts

```diff
@@ -0,0 +1,10 @@
+import { Page } from 'puppeteer'
+
+/**
+ * Get the thought value that cursor on.
+ */
+const getEditingText = (page: Page) => page.evaluateHandle(() => {
+  return document.querySelector('.editing .editable')?.innerHTML
+})
```

Comment:
You can use `page.evaluate` instead. That way it will simply return a string instead of `JSHandle` and you don't have to access `_remoteObject.value`

Reply:
You're right! Thanks.